package.json

```diff
@@ -66,5 +66,8 @@
         "typescript": "4.6.4",
         "use-debounce": "^8.0.1",
         "wagmi": "^0.5.12"
+    },
+    "dependencies": {
+        "numbro": "^2.3.6"
```

Comment:
when you do add dependencies, make sure you add them as dev dependencies, reference all of the other dependencies in this file.

Reply:
noted